embeds.xml

```diff
@@ -0,0 +1,9 @@
+<Ui xsi:schemaLocation="http://www.blizzard.com/wow/ui/ ..\FrameXML\UI.xsd">
+  <Script file="Libs\LibStub\LibStub.lua"/>
+  <Include file="Libs\CallbackHandler-1.0\CallbackHandler-1.0.xml"/>
+  <Include file="Libs\LibDataBroker-1.1\LibDataBroker-1.1.lua"/>
+  <Include file="Libs\AceAddon-3.0\AceAddon-3.0.xml"/>
+  <Include file="Libs\AceGUI-3.0\AceGUI-3.0.xml"/>
+  <Include file="Libs\LibDBIcon-1.0\LibDBIcon-1.0\lib.xml"/>
+  <Include file="Libs\ChatThrottleLib.lua"/>
```

Comment:
Where was this being loaded before?

Reply:
ChatThrottleLib was loaded in Hardcore.toc.
I changed it to use embeds.xml since it is apparently a standard used for WoW addons to load libraries in a embeds.xml file.

Reply:
Embeds are BiS